backend/src/test/java/de/sambam/gamenighttracker/service/PlayerServiceTest.java

```diff
@@ -36,4 +36,22 @@ public void addPlayerTester() {
         assertTrue(playerList.contains(Player.builder().id("003").name("Thomas").build()));
 
     }
+
+    @Test
+    @DisplayName("listAllPlayers should return all players")
```

Comment:
you say that you want to test whether the method returns something, but in your //THEN block you only check if something is in your database

Reply:
thanks for the hint. just made the respective changes ^^